homeassistant/components/yamaha/media_player.py

```diff
@@ -132,7 +132,7 @@ def _discovery(config_info):
         zones = None
 
         # Fix for upstream issues in rxv.find() with some hardware.
-        with contextlib.suppress(AttributeError):
+        with contextlib.suppress(AttributeError, UnicodeDecodeError, ValueError):
```

Comment:
`UnicodeDecodeError` is a subclass of `ValueError`.
```suggestion
        with contextlib.suppress(AttributeError, ValueError):
```